dot/parachain/candidate-validation/candidate_validation_test.go

```diff
@@ -677,3 +669,215 @@ func Test_performBasicChecks(t *testing.T) {
+		MaxPovSize:             uint32(2048),
+	}
+
+	expectedPersistedValidationDataSmallMax := parachaintypes.PersistedValidationData{
+		ParentHead:             parachaintypes.HeadData{Data: hd},
+		RelayParentNumber:      uint32(1),
+		RelayParentStorageRoot: common.MustHexToHash("0x50c969706800c0e9c3c4565dc2babb25e4a73d1db0dee1bcf7745535a32e7ca1"),
+		MaxPovSize:             uint32(10),
+	}
+
+	mockInstance := NewMockRuntimeInstance(ctrl)
+	mockInstance.EXPECT().
+		ParachainHostPersistedValidationData(
+			uint32(1000),
+			gomock.AssignableToTypeOf(parachaintypes.OccupiedCoreAssumption{})).
+		Return(&expectedPersistedValidationData, nil).AnyTimes()
+	mockInstance.EXPECT().
+		ParachainHostValidationCode(uint32(1000), gomock.AssignableToTypeOf(parachaintypes.OccupiedCoreAssumption{})).
+		Return(&validationCode, nil).AnyTimes()
+
+	mockInstance.EXPECT().
+		ParachainHostPersistedValidationData(
+			uint32(2),
+			gomock.AssignableToTypeOf(parachaintypes.OccupiedCoreAssumption{})).
+		Return(&expectedPersistedValidationData, nil).AnyTimes()
+	mockInstance.EXPECT().
+		ParachainHostValidationCode(uint32(2), gomock.AssignableToTypeOf(parachaintypes.OccupiedCoreAssumption{})).
+		Return(&validationCode, nil).AnyTimes()
+
+	mockInstance.EXPECT().
+		ParachainHostPersistedValidationData(
+			uint32(3),
+			gomock.AssignableToTypeOf(parachaintypes.OccupiedCoreAssumption{})).
+		Return(&expectedPersistedValidationDataSmallMax, nil).AnyTimes()
+	mockInstance.EXPECT().
+		ParachainHostValidationCode(uint32(3), gomock.AssignableToTypeOf(parachaintypes.OccupiedCoreAssumption{})).
+		Return(&validationCode, nil).AnyTimes()
+
+	mockInstance.EXPECT().
+		ParachainHostPersistedValidationData(
+			uint32(4),
+			gomock.AssignableToTypeOf(parachaintypes.OccupiedCoreAssumption{})).
+		Return(&expectedPersistedValidationData, nil).AnyTimes()
+	mockInstance.EXPECT().
+		ParachainHostValidationCode(uint32(4), gomock.AssignableToTypeOf(parachaintypes.OccupiedCoreAssumption{})).
+		Return(&validationCode, nil).AnyTimes()
+
+	mockInstance.EXPECT().
+		ParachainHostPersistedValidationData(
+			uint32(5),
+			gomock.AssignableToTypeOf(parachaintypes.OccupiedCoreAssumption{})).
+		Return(&expectedPersistedValidationData, nil).AnyTimes()
+	mockInstance.EXPECT().
+		ParachainHostValidationCode(uint32(5), gomock.AssignableToTypeOf(parachaintypes.OccupiedCoreAssumption{})).
+		Return(&validationCode, nil).AnyTimes()
+
+	bd, err := scale.Marshal(BlockDataInAdderParachain{
+		State: uint64(1),
+		Add:   uint64(1),
+	})
+	require.NoError(t, err)
+	pov := parachaintypes.PoV{
+		BlockData: bd,
+	}
+
+	type args struct {
+		runtimeInstance  parachainruntime.RuntimeInstance
+		candidateReceipt parachaintypes.CandidateReceipt
+		pov              parachaintypes.PoV
+	}
+	tests := map[string]struct {
+		args          args
+		want          *ValidationResult
+		expectedError error
+		isValid       bool
+	}{
+		"invalid_pov_hash": {
+			args: args{
+				runtimeInstance:  mockInstance,
```

Comment:
you are passing the same mock instance everywhere. If it is only one value, you can just use a common mockInstance for all tests.
pov is also just one value. 

So, no need to pass them as arguments

Reply:
Good point, I've updated this.